src/Compute/Compute/help/Set-AzVMADDomainExtension.md

```diff
@@ -28,7 +28,12 @@ This extension lets your virtual machine join a domain.
 
 ### Example 1
 ```powershell
-Set-AzVMADDomainExtension -ResourceGroupName "ResourceGroup11" -VMName "VirtualMachine07" -DomainName "abc.com"
+# Create a Resource Group and Virtual Machine before this.
+$extensionName = "extensionName";
+$virtualMachineName = "vmName";
+$resourceGroupName = "resourceGroupName";
+$domainName = "domain.com";
+Set-AzVMADDomainExtension -ResourceGroupName $resourceGroupName -VMName $virtualMachineName -Name $extensionName -DomainName $domainName;
```

Comment:
```suggestion
$extensionName = "extensionName"
$virtualMachineName = "vmName"
$resourceGroupName = "resourceGroupName"
$domainName = "domain.com"
Set-AzVMADDomainExtension -ResourceGroupName $resourceGroupName -VMName $virtualMachineName -Name $extensionName -DomainName $domainName
```
PowerShell script doesn't need to end with semicolon

Reply:
@BethanyZhou , I know it doesn't need to, but I think it makes the lines more clear to the reader. 
Is there an official Best Practice about not using Semicolons?

Reply:
No. But in our conventions, we don't add semicolon in the end of line.